deploy/irdb_clowdapp.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ccx-data-pipeline
+objects:
+
+- kind: HorizontalPodAutoscaler
+  apiVersion: autoscaling/v1
+  metadata:
+    labels:
+      app: ccx-data-pipeline
+    name: ccx-data-pipeline
+  spec:
+    minReplicas: ${{MIN_REPLICAS}}
+    maxReplicas: ${{MAX_REPLICAS}}
+    scaleTargetRef:
+      apiVersion: v1
+      kind: Deployment
+      name: ccx-data-pipeline-results-db-writer
+    targetCPUUtilizationPercentage: 50
+
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ccx-data-pipeline
+  spec:
+    envName: ${ENV_NAME}
+    dependencies:
+      - ingress
+    deployments:
+      - name: results-db-writer
+        minReplicas: ${{MIN_REPLICAS}}
+        webServices:
+          public:
+            enabled: false
+          private:
+            enabled: false
+          metrics:
+            enabled: true
+        podSpec:
+          env:
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__ENABLED
+              value: "false"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__METRICS__ENABLED
+              value: "false"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__DB_DRIVER
+              value: postgres
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__PG_PARAMS
+              value: sslmode=disable
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__PG_HOST
+              valueFrom:
+                secretKeyRef:
+                  key: db.host
+                  name: "${DATABASE_SERVICE_NAME}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__PG_DB_NAME
+              valueFrom:
+                secretKeyRef:
+                  key: db.name
+                  name: "${DATABASE_SERVICE_NAME}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__PG_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  key: db.user
+                  name: "${DATABASE_SERVICE_NAME}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__PG_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: db.password
+                  name: "${DATABASE_SERVICE_NAME}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__STORAGE__PG_PORT
+              valueFrom:
+                secretKeyRef:
+                  key: db.port
+                  name: "${DATABASE_SERVICE_NAME}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__LOGGING__LOGGING_TO_CLOUD_WATCH_ENABLED
+              value: "false"
+          image: ${IMAGE}:${IMAGE_TAG}
+          resources:
+            requests:
+              cpu: 100m
+              memory: 150Mi
+            limits:
+              cpu: 200m
+              memory: 300Mi
+          command:
+              - ./insights-results-aggregator
+              - migration
+              - latest
```

Comment:
This command is meant for 'initcontainers' section - to run DB migration once before the pod starts. Then it should just execute the default cmd/entry point in the image. Is there anything like "initcontainers" for clowder syntax?
https://docs.openshift.com/container-platform/4.1/nodes/containers/nodes-containers-init.html

Reply:
@matysek yes there is https://redhatinsights.github.io/clowder/clowder/dev/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-podspec